src/stretchy.js

```diff
@@ -54,11 +54,15 @@ const stretchyCodePoint: {[string]: string} = {
     xrightleftarrows: "\u21c4",
     xrightequilibrium: "\u21cc",  // Not a perfect match.
     xleftequilibrium: "\u21cb",   // None better available.
+    cdrightarrow: "\u2192",
+    cdleftarrow: "\u2190",
+    cdlongequal: "=",
 };
 
 const mathMLnode = function(label: string): mathMLTree.MathNode {
+    const key = label.replace(/^\\+/, "");
```

Comment:
I think we could run into an issue here if someone defines a macro like `\\cdrightarrow` which wouldn't overwrite `\\\\cdrightarrow`, but both would have the same `key` here.

Reply:
Ah yes, I didn't notice that. Fixed now.